test/backend_test.in.cpp

```diff
@@ -9040,3 +9036,153 @@ NGRAPH_TEST(${BACKEND_NAME}, argmax_trivial)
     backend->call_with_validate(f, {result}, {a});
     EXPECT_EQ((vector<int>{1, 3, 0}), read_vector<int>(result));
 }
+
```

Comment:
Could you do some trivial tests on 1D and 3D tensors as well?

Also, we should add some checks on the validation stuff in `type_prop.cpp`.

Reply:
Added 1D, 3D tests. Added validation tests in type_prop.cpp